src/System.Private.Xml/tests/XmlSerializer/System.Xml.XmlSerializer.Tests.settings.targets

```diff
@@ -6,7 +6,7 @@
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
     <OutputType>Library</OutputType>
     <AppDesignerFolder>Properties</AppDesignerFolder>
-    <DefineConstants Condition="'$(TargetGroup)' == 'netcore50' OR '$(TargetGroup)' == 'netcore50aot'">$(DefineConstants);NET_NATIVE</DefineConstants>
+    <DefineConstants Condition="'$(TargetGroup)' == 'uapaot'">$(DefineConstants);uapaot</DefineConstants>
```

Comment:
> None of this will kick in today because of there is no uap or uapaot build configuration in https://github.com/dotnet/corefx/blob/master/src/System.Private.Xml/tests/XmlSerializer/Configurations.props

Understood, but I suppose we will enable them one day in the future. Here we just keep the same logic of what they were.